src/main/java/com/mitchej123/hodgepodge/mixins/late/extrautilities/MixinItemHealingAxe.java

```diff
@@ -0,0 +1,50 @@
+package com.mitchej123.hodgepodge.mixins.late.extrautilities;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityLiving;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.DamageSource;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import com.llamalad7.mixinextras.sugar.Local;
+import com.rwtema.extrautils.item.ItemHealingAxe;
+
+@Mixin(ItemHealingAxe.class)
+abstract public class MixinItemHealingAxe {
+
+    // The original method uses `getHealth() < getHealth()` which is the cause of the bug,
+    // Replace the second getHealth call in the if statement with 0. (0 < getHealth())
+    @Redirect(
+            method = "onLeftClickEntity",
+            at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/EntityLiving;getHealth()F", ordinal = 1))
+    private float hodgepodge$redirectGetHealth(EntityLiving instance) {
+        return 0F;
+    }
+
+    // Cancel the hunger reduction
+    @Redirect(
+            method = "onLeftClickEntity",
+            at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/player/EntityPlayer;addExhaustion(F)V"))
+    private void hodgepodge$redirectAddExhaustion(EntityPlayer player, float p_71020_1_) {
+
+    }
```

Comment:
Regardless of original intention, I think the hunger reduction from healing axe is actually an accidentally useful thing, as it provides a convenient way to drain your hunger if you are trying to eat many new foods for something like spice of life heart gain

Reply:
Agreed, that should stay.
While holding it, it gives you hunger, so while using it, it should drain your hunger.

Reply:
> I think the hunger reduction from healing axe is actually an accidentally useful thing, as it provides a convenient way to drain your hunger

Yeah true, it also makes sense how much the lack of hunger would mess you up if you don't kill your character while healing a mob and won't let you immediately naturally heal yourself back. I'll add it back